src/ucp/core/ucp_mm.c

```diff
@@ -1038,3 +1042,23 @@ void ucp_mem_rcache_cleanup(ucp_context_h context)
         ucs_rcache_destroy(context->rcache);
     }
 }
+
+UCS_STATIC_INIT {
+    ucp_md_index_t md_index;
+
+    ucp_mem_dummy_handle = ucs_malloc(sizeof(ucp_mem_t) +
+                                      sizeof(uct_mem_h) * UCP_MAX_MDS,
```

Comment:
can we avoid the static init (which is problem for static link) and keep ucp_mem_dummy_handle as a global struct with UCP_MAX_MDS array of uct_mem_h?